src/utils/order-tag.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Tag } from "@carbon/react";
+
+export const OrderTag = (order: any) => {
+  if (
+    (order?.action === "NEW" || order?.action === "REVISE") &&
+    order.dateStopped === null
+  ) {
+    return (
+      <Tag
+        style={{
+          background: "#6F6F6F",
+          color: "white",
```

Comment:
This component can be cleaned to return only the color code that is passed here like https://github.com/openmrs/openmrs-esm-patient-chart/blob/feat/lab-results/packages/esm-patient-orders-app/src/utils/utils.ts#L14  and used like this https://github.com/openmrs/openmrs-esm-patient-chart/blob/38b995cea1a77185da7b0b4c27ae6b4589981f7d/packages/esm-patient-orders-app/src/components/orders-details-table.component.tsx#L150